applications/nrf5340_audio/src/bluetooth/le_audio.c

```diff
@@ -0,0 +1,253 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include "le_audio.h"
+
+#include <zephyr/zbus/zbus.h>
+#include <zephyr/bluetooth/conn.h>
+
+#include "nrf5340_audio_common.h"
+#include "bt_mgmt.h"
+#include "bt_rend.h"
+#include "bt_content_ctrl.h"
+#include "broadcast_internal.h"
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(le_audio, CONFIG_BLE_LOG_LEVEL);
+
+static enum le_audio_device_type device_type;
+
+/**
+ * @brief	Zbus listener to receive events from bt_mgmt
+ *
+ * @note	Will in most cases be called from BT_RX context,
+ *		so there should not be too much processing done here
+ */
+static void bt_mgmt_evt_handler(const struct zbus_channel *chan)
+{
+	int ret;
+	const struct bt_mgmt_msg *msg;
+
+	msg = zbus_chan_const_msg(chan);
+	uint8_t event = msg->event;
+
+	switch (event) {
+	case BT_MGMT_EXT_ADV_READY:
+		LOG_INF("Ext adv ready");
+		if (IS_ENABLED(CONFIG_TRANSPORT_BIS)) {
+			ret = broadcast_source_start(msg->ext_adv);
+			if (ret) {
+				LOG_ERR("Failed to start broadcaster: %d", ret);
```

Comment:
```suggestion
				LOG_ERR("Failed to start broadcast source: %d", ret);
```

Replace 'broadcaster' -> 'broadcast source' every place in this file

Reply:
But then we are reducing it from a device type to an audio role